Formula/notmuch.rb

```diff
@@ -1,8 +1,8 @@
 class Notmuch < Formula
   desc "Thread-based email index, search, and tagging"
   homepage "https://notmuchmail.org"
-  url "https://notmuchmail.org/releases/notmuch-0.23.7.tar.gz"
-  sha256 "f11bb10d71945f6c3f16d23117afc70810aa485878e66bb4bf43cc3f08038913"
+  url "https://notmuchmail.org/releases/notmuch-0.24.tar.gz"
```

Comment:
If upstream doesn't accept the patch, we shouldn't be carrying it anyway, as I understand our policy.